### ISSUE TITLE
Streams resolution parameter default value

### ISSUE THREAD

**Issue by qligier** [basvandorst/StravaPHP#17]
Hey,
The default value of the `resolution` parameter in the streams function is set to `all`:
```php
$client->getStreamsActivity($id, $types, $resolution = 'all', $series_type = 'distance');
$client->getStreamsEffort($id, $types, $resolution = 'all', $series_type = 'distance');
$client->getStreamsSegment($id, $types, $resolution = 'all', $series_type = 'distance');
```
[The documentation](http://strava.github.io/api/v3/streams/) isn't quite clear on this, but `all` isn't a valid value:
```bash
curl -G https://www.strava.com/api/v3/activities/683597852/streams/latlng -H "Authorization: Bearer abc" -d resolution=all
```
returns
```json
{"message":"Bad Request","errors":[{"resource":"Application","field":"resolution","code":"invalid"}]}
```

As I understand it:
If you don't specify the parameter, all points are returned.
If you specify it (with `low`, `medium` or `high`), only (100/1000/10000) values are returned.

**Comment by fbonzon** [comment]
I confirm. To mean "all" the `resolution` should be left empty. I.e. replace everywhere `'all'` by the empty string `''`.

**Comment by qligier** [comment]
Setting the parameter to null also works, that's what I've done to [patch this issue](https://github.com/qligier/StravaPHP/commit/5c72d615d33ef3191c7a76551132c7291795f4df) in my local repository.

**Comment by fbonzon** [comment]
Correct, `null` works too.

Could you change the 3 occurrences of `'all'` to `null` in those 3 files too in your fork:
src/Strava/API/Service/REST.php
src/Strava/API/Service/ServiceInterface.php
src/Strava/API/Service/Stub.php
and send a PR here?

**Comment by qligier** [comment]
Done. The code works but it isn't passing the tests because of the new versions of dependencies. I'll fix that later.

**Comment by basvandorst** [comment]
Fixed in 1.0.2. Thanks to @qligier
